### PR TITLE
[skip appveyor]

### PR DESCRIPTION
**Pull request by ocefpaf**
Should close https://github.com/conda-forge/conda-forge.github.io/issues/45 (Which I just realized I opened in the wrong repo.)

@pelson can you take a look at this?

### CODE REVIEW

File: .CI/create_feedstocks.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def write_token(name, token):
     if removed_recipes:
         subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
         msg = ('Removed recipe{s} ({}) after converting into feedstock{s}.'
```
Comment (jakirkham) [inline .CI/create_feedstocks.py]: Wouldn't you want `'[skip appveyor]'` at the end of this line?

Comment (ocefpaf) [inline .CI/create_feedstocks.py]: I guess it reads better in the next line. I just need to add a space.  Not sure if `feedstocks.[skip appveyor]` will be understood.

Comment (jakirkham) [inline .CI/create_feedstocks.py]: Yes, definitely a space.

**Comment by jakirkham** [comment]
Looks good. Sorry for the bikeshedding.

**Comment by ocefpaf** [comment]
> Sorry for the bikeshedding.

What would be of GitHub issues without bikeshedding!  Madness!! :stuck_out_tongue_winking_eye:

File: .CI/create_feedstocks.py [hunk 2]
```diff
@@ -137,8 +137,8 @@ def write_token(name, token):
     subprocess.check_call(['git', 'status'])
     if removed_recipes:
         subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
-        msg = ('Removed recipe{s} ({}) after converting into feedstock{s}.'
-               ''.format(', '.join(removed_recipes),
+        msg = ('Removed recipe{s} ({}) after converting into feedstock{s}. '
+               '[skip appveyor]'.format(', '.join(removed_recipes),
```
Comment (pelson) [inline .CI/create_feedstocks.py]: I wonder if we can just skip all CI? I don't think we need to do anything after this commit has been done.

Comment (ocefpaf) [inline .CI/create_feedstocks.py]: @jakirkham what is the syntax to skip all? [`skip ci`]?

**Comment by ocefpaf** [comment]
`[skip ci]` seems to skip both AppVeyor and Travis-CI, but CircleCI is still going.

File: .CI/create_feedstocks.py (hunk 2 above)
Comment (jakirkham) [inline .CI/create_feedstocks.py]: All of them should take `[ci skip]`. I think Travis will not take `[skip ci]` yet.

**Comment by jakirkham** [comment]
Weird. Their [docs]( https://circleci.com/docs/skip-a-build ) say the should.

**Comment by jakirkham** [comment]
Maybe try `[ci skip]`. That has been [around longer]( https://discuss.circleci.com/t/ci-skip-is-great-but-it-is-backwards/418 ).

File: .CI/create_feedstocks.py (hunk 2 above)
Comment (jakirkham) [inline .CI/create_feedstocks.py]: I somehow missed this is only for removals. :smile: Definitely, let's skip them all.

**Comment by ocefpaf** [comment]
> Maybe try [ci skip]. That has been around longer.

Nope. CircleCI is still starting.

**Comment by jakirkham** [comment]
Ok, sorry didn't look at the Circle CI build.

This is actually how they do skips. Basically, they start the build and then skip doing anything after they clone and see the note in the commit.

I agree it is kind of annoying that it isn't a true skip.  There might be a discussion on Circle CI about this. Though Circle CI is our fastest build queue so I am less worried about this. Getting it removed from AppVeyor and Travis CI is already a big win.

I am +1 on merging this.

**Comment by ocefpaf** [comment]
> Though Circle CI is our fastest build queue so I am less worried about this.

Agreed.

Ping @pelson are you OK with this?

**Comment by ocefpaf** [comment]
Ping @pelson. We are waiting for your comments on this one.

**Comment by pelson** [comment]
> Ping @pelson. We are waiting for your comments on this one.

It's the weekend @ocefpaf! :wink: 

Sorry it has taken me a while to get to this issue.

**Comment by ocefpaf** [comment]
> It's the weekend @ocefpaf! :wink:

You don't have to answer :wink: 

(I only ping you due to my cowardliness to merge important changes.)

**Comment by jakirkham** [comment]
I should also add I have seen sometimes CI runs a PR anyways because the merge commit doesn't say `[ci skip]`. So, that can also have an effect. Adding it to the title of the PR has normally solved that for me, but that would have been antithetical here.

**Pull request merged**